### DISCUSSION TITLE
why /performance not working?? crashing

### DISCUSSION THREAD

**Discussion by ArnavK-09** [browserless/chrome#3992] [Q&A]
![image](https://github.com/browserless/browserless/assets/69188140/b5bf9606-06e5-4f0f-9b1b-ed9928733571)

**Comment by joelgriffith** [comment]
What URL are you using?

**Reply by ArnavK-09** [reply]
For browserless: POST https://<browserless docker server url>/performance

Data: Url: https://example.com

**Reply by joelgriffith** [reply]
What's the size of machine you're executing this on? Example.com is pretty light so it should work rather well

**Reply by ArnavK-09** [reply]
Was using in GitHub codespaces base

**Reply by joelgriffith** [reply]
I wonder if GitHub codespaces don't allow child processes given that ESRCH error. We do explicitly open lighthouse in a child process due to the fact that it can only operate 1-at-a-time per NodeJS process
